pkg/addons/data/ClusterRoleBinding.rbac.authorization.k8s.io/service-catalog-controller-group-sync-binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: service-catalog-controller-group-sync-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: group-sync
+subjects:
+- kind: ServiceAccount
+  name: service-catalog-controller
+  namespace: kube-service-catalog
```

Comment:
why all the references to service catalog? that's definitely not right.  At the moment the customer admin controller runs in the openshift-infra namespace under the customer-admin-controller service account.

Reply:
ok, makes sense - I did this ages ago